.github/workflows/build-preview.yml

```
@@ -0,0 +1,36 @@
+name: Build PR Preview
+on:
+  pull_request:
+    branches: [main]
+    types:
+      - opened
+      - reopened
+      - synchronize
```

```suggestion
      - synchronize
    - workflow_dispatch
```
If you add `workflow_dispatch` here you'll get a little dropdown on the actions page which lets you run the action on command. If you wanted to test that way. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow

You could also remove the`branches` filter, that should run it too.

---

Commented on the first action but this applies to whatever you want to test.